pandas/tests/window/test_groupby.py

```diff
@@ -897,10 +897,11 @@ def test_as_index_false(self, by, expected_data):
         )
         tm.assert_frame_equal(result, expected)
 
-    def test_nan_and_zero_endpoints(self):
+    def test_nan_and_zero_endpoints(self, any_int_numpy_dtype):
         # https://github.com/twosigma/pandas/issues/53
+        dtype = np.dtype(any_int_numpy_dtype).type
```

Comment:
can we call this something other than dtype, since it isnt one

Reply:
Ok, I can call it `typ`.